client/app/models/milestone/constants.js

```diff
@@ -58,6 +62,7 @@ export const REVIEW_MILESTONE_IDS = [
 export const UPCOMING_MILESTONES = [
```

Comment:
This UPCOMING_MILESTONES constant is references in MILESTONE_LIST_BY_TAB_LOOKUP, which is imported into the assignment section. So, this could affect that portion of the app, but it seems to be okay afaict.